backend/forecastbox/config.py

```diff
@@ -177,3 +227,6 @@ def validate_runtime(config: FIABConfig) -> None:
 
 
 config = FIABConfig()
+config.save_example_to_file()
```

Comment:
on every import of this module this would be triggered? So runs of mypy, pytest, etc, ... all would trigger a save to the possibly non-existent ~/.fiab? Thats sounds unhealthy